### DISCUSSION TITLE
Allow uploading with multiple date formats in the WorkBench

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5616] [Feature Requests]
The WorkBench currently requires the uploader to have the dates conform to specifically the local date format (MM/DD/YYYY or similar). 

Many dates are stored in fields that look like the following:
`November 1999`
`13 July 2004`
`4/1999`
`Dec 1 2010`

The WorkBench should be intelligent enough to know what these mean and interpret these.

**Comment by maxpatiiuk** [comment]
Perhaps a better solution would be to add a date converter to the workbench.

The reason it is better is because the date converter can highlight any invalid dates even before you try to validate the dataset, and the date converter would allow you see what the converted date would look like.

In contrast, if the conversion is done on the back-end, the value saved in the record, and the value shown in the data set won't match leading to possible confusion

**Comment by grantfitzsimmons** [comment]
Christian at the Gothenburg Museum of Natural History requested this as well.

Importantly, we should support stripping datetime values to just the format accepted by the WorkBench

![image](https://user-images.githubusercontent.com/37256050/201702821-04942bd4-07aa-41cb-ac09-cfe48777656b.png)

**Comment by maxpatiiuk** [comment]
This might have been partially fixed in https://github.com/specify/specify7/pull/2857 as workbench import has been reworked to convert some date formats to the date format set in remote prefs

**Comment by grantfitzsimmons** [comment]
This has been requested by Heather Cole at AAFC

> I would like the workbench to accept multiple different date formats to be acceptable in the workbench space, I thought that is how it usually worked, but it seems like setting a system format maybe over-wrote that?
